packages/render-fidelity-tools/src/components/renderers/dspbr-pt-viewer.ts

```diff
@@ -80,29 +80,29 @@ export class PathtracingViewer extends LitElement {
     this[$renderer] = new PathtracingRenderer(this[$canvas]!, enableControls);
 
     const renderer = this[$renderer];
-    renderer.autoScaleOnImport(false);
```

Comment:
At some point I mentioned that, by default, the renderer normalizes scene dimensions on import. This yields a more predictable input scenario for the raytracing kernel and prevents numerical issues. That behavior was explicitly disabled in the previous version because camera transformations were not scaled accordingly. The new version adds this. This should help with robustness, but I didn't test it excessively.